### PR TITLE
Use jsdeliver cdn

### PR DESCRIPTION
**Pull request by icyleaf**
#6

### CODE REVIEW

File: src/swagger/http/views/oauth2-redirect.html [hunk 1]
```diff
@@ -65,4 +65,4 @@
         }
         window.close();
     }
-</script>
+</script>
```
Comment (j8r) [inline src/swagger/http/views/oauth2-redirect.html]: Missing newline :P

File: src/swagger/http/views/swagger.ecr [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <head>
     <meta charset="UTF-8">
     <title><%= title %></title>
-    <link rel="stylesheet" type="text/css" href="/swagger-ui/swagger-ui.css" >
+    <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/swagger-ui-dist@3/swagger-ui.css" >
     <link rel="icon" type="image/png" href="/swagger-ui/favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="/swagger-ui/favicon-16x16.png" sizes="16x16" />
```
Comment (j8r) [inline src/swagger/http/views/swagger.ecr]: The icons are also available in the CDN.

File: src/swagger/http/views/oauth2-redirect.html (hunk 1 above)
Comment (icyleaf) [reply src/swagger/http/views/oauth2-redirect.html]: Updated

File: src/swagger/http/views/swagger.ecr (hunk 2 above)
Comment (icyleaf) [reply src/swagger/http/views/swagger.ecr]: You got me.

**Review by j8r — APPROVED** [review]

**Pull request merged**